packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/BasicLoginCardLayout.js

```diff
@@ -11,7 +11,9 @@ const BasicLoginCardLayout = ({ children, layout, ...props }) => (
 );
 
 BasicLoginCardLayout.propTypes = {
+  /** Children nodes. */
   children: PropTypes.node,
+  /** Additional layout props to override the default ones. */
   layout: PropTypes.object
 };
```

Comment:
unfortunately because we started down the path of using `layout` we need to leave it in so line 7 would look something like ...
```
<Col sm={10} smOffset={1} md={8} mdOffset={2} lg={8} lgOffset={2} {...layout} {...props}>
```
... but this pseudo code needs to be double checked

Reply:
Do you feel there's a chance someone used the component directly?

Reply:
I don't believe so, and I really think this change is for good,
And helps to avoid future override errors.